lib/licenses.nix

```diff
@@ -387,6 +407,11 @@ in mkLicense lset) ({
     fullName  = "CeCILL-C Free Software License Agreement";
   };
 
+  cronyx = {
+    spdxId = "Cronyx";
+    fullName = "Cronyx License";
+  };
```

Comment:
```
This software may be used, modified, copied, distributed, and sold,
in both source and binary form provided that the above copyright
and these terms are retained. Under no circumstances is the author
responsible for the proper functioning of this software, nor does
the author assume any responsibility for damages incurred with its use.
```

Seems obviously FOSS?

Reply:
Oh, I guess `free = true;` is the default? Sorry!

Reply:
yes i only included `free = true` with an explaination text for licenses that have debateable freeness

Reply:
yes i only included `free = true` with an explaination text for licenses that have debateable freeness

Reply:
yes i only included `free = true` with an explaination text for licenses that have debateable freeness

Reply:
my github phone app told me that there was an error while posting my comment, so i retried a few times resulting 5 comments (i deleted 4 of them now)